rmw_fastrtps_dynamic_cpp/Doxyfile

```diff
@@ -0,0 +1,31 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rmw_fastrtps_dynamic_cpp"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Implement the ROS middleware interface using eProsima FastRTPS  static code generation in C++."
```

Comment:
It looks like the package.xml is incorrect, but this should be "dynamic code generation"